packages/app-extensions/src/rest/rest.js

```diff
@@ -42,15 +44,30 @@ export const setNullBusinessUnit = value => {
  */
 export function* requestSaga(resource, options = {}) {
   const requestData = yield call(prepareRequest, resource, options)
-  let response = yield call(
-    sendRequest,
-    requestData.url,
-    requestData.options,
-    options.acceptedErrorCodes,
-    options.acceptedStatusCodes
-  )
-  response = yield call(handleClientQuestion, response, requestData, options)
-  return response
+  try {
+    let response = yield call(
+      sendRequest,
+      requestData.url,
+      requestData.options,
+      options.acceptedErrorCodes,
+      options.acceptedStatusCodes
+    )
+    response = yield call(handleClientQuestion, response, requestData, options)
+    return response
+  } catch (error) {
+    if (error instanceof InformationError) {
```

Comment:
As far as i understand this is the fallback handling. Would move it to a separate function and set a function name that suggest, that its a fallback.